source/_integrations/velux.markdown

```diff
@@ -67,6 +67,31 @@ automation:
     - service: velux.reboot_gateway
 ```
 
+## Attributes
+
+Cover entity with integrated rain sensor has extra attributes to represent the state of the limitation sensor.
+
+| Name | Description |
+| ---- | ----------- |
+| `limitation_min` | Min limitation of window. `limitation_min` => 0 no rain; `limitation_min` => 93 the window can only open for 7 % , so it means, its raining
+| `limitation_max` | Max limitation of window.
+
+The value can used as a `binary_sensor` for rain detection `True` or no rain `False`
```

Comment:
```suggestion
The value can be used as a `binary_sensor` for rain detection `True` or no rain `False`
```

Reply:
Fixed in [3c19ffb](https://github.com/home-assistant/home-assistant.io/pull/21486/commits/3c19ffb1e5e889e3fa81cc7b16e75bfbc99f6dc6)